app/controllers/parameters_controller.rb

```diff
@@ -2,29 +2,43 @@
 
 class ParametersController < ApplicationController
   before_action :find_parameter, only: %i[update destroy]
+  before_action :find_parameter, only: %i[update destroy]
+  before_action :find_parameter, only: %i[update destroy]
 
   def create
     @parameter = Parameter.new(parameter_params)
-
-    @parameter.save!
-
-    render json: @parameter
+    if @parameter.save
+      update_last_edited_by
+      render json: @parameter
+    else
+      render500
+    end
   end
 
   def update
-    @parameter.update(parameter_params)
-
-    render json: @parameter
+    if @parameter.update(parameter_params)
+      update_last_edited_by
```

Comment:
Sorry if this is a dumb question, but if the last updated by id is added to the params, why do you need to call this method after it is updated?

Reply:
That's now clearer with the refactoring done

Reply:
I think I was more wondering why sometimes you are adding the last updater in the params, and sometimes by manually calling `update_last_edited_by`.  Do you need both strategies? Or am I misunderstanding the difference between them?